src/connector/src/source/cdc/external/mod.rs

```diff
@@ -250,7 +249,7 @@ pub struct ExternalTableConfig {
     #[serde(rename = "table.name")]
     pub table: String,
     /// `ssl.mode` specifies the SSL/TLS encryption level for secure communication with Postgres.
-    /// Choices include `disable`, `prefer`, and `require`.
+    /// Choices include `disabled`, `preferred`, and `required`.
```

Comment:
Just an FYI: 😄 

disable/prefer/require actually uses the naming convention from PG, although it may loo
<img width="1354" alt="Screenshot 2024-05-05 at 19 20 29" src="https://github.com/risingwavelabs/risingwave/assets/6970676/cc2c2d37-438d-4b96-ad33-0e8a2c777c21">
k like a grammar mistake.

Reply:
Yeah, I know. It is just a interface to the end users, and mysql uses `disabled`, `preferred`, and `required` which have same semantic as PG. So I decide to unify the naming.

Reply:
This is a breaking change...